alpine.expect

```diff
@@ -0,0 +1,53 @@
+set timeout 300
+
+spawn sudo kvm -m 4G -smp 2 -drive file=alpine-virt.iso,format=raw,media=cdrom -drive file=rootfs.qcow2 -nographic
```

Comment:
Do you know why this doesn't work for me locally? The VM boots fine but `ping google.com` doesn't work after `ifup eth0` step.